ground/src/main/java/com/google/android/ground/ui/map/gms/ConverterExt.kt

```diff
@@ -38,3 +38,7 @@ fun Coordinates.toLatLng(): LatLng = LatLng(lat, lng)
 fun List<Coordinates>.toLatLngList(): List<LatLng> = map { it.toLatLng() }
 
 fun Location.toCoordinates(): Coordinates = Coordinates(latitude, longitude)
+
+fun Location.toAltitude(): Double? = if (hasAltitude()) altitude else null
```

Comment:
These new "to" methods seem like misnomers. Maybe `getAltitudeOrNull()? and `getAccuracyOrNull()` would more correct and kotlinic?